fdm-app/app/components/custom/atlas/atlas-panels.tsx

```
@@ -17,6 +17,13 @@ import {
     CardTitle,
 } from "~/components/ui/card"
 import { cn } from "~/lib/utils"
+import type { FeatureCollection } from "geojson"
```

Biome (1.9.4)

[error] Shouldn't redeclare 'FeatureCollection'. Consider to delete it or rename it.

'FeatureCollection' is defined here:


(lint/suspicious/noRedeclare)